refactor : EmotionRanking 마크업 접근성 개선 리팩토링

gmlyb1 opened the pull request:
- 마크업 접근성 개선 refactor

### 제목 :[refactor : EmotionRanking 마크업 접근성 개선 리팩토링]

### PR 유형

- [ ] 새로운 기능 추가
- [ ] 버그 수정
- [ ] CSS 등 사용자 UI 디자인 변경
- [ ] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [x] 코드 리팩토링
- [ ] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [ ] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

### PR 체크리스트

<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->

- [ ] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [ ] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### 🔎 작업 내용

<!-- 어떤 작업을 하셨는지 상세하게 작성해주세요-->

- EmotionRanking 마크업 접근성 개선 refactor 작업
- text # 55555 등 tailwind 에 맞는 css 코드 수정
- aria-label, aria-labelledby 추가

  <br />

### 🔧 앞으로의 과제

- 내일 할 일을 적어주세요
  <br />

### 이슈

<!-- 이슈 키워드와 함께 #을 입력한 후 이슈 번호를 선택해주세요. -->
<!-- 에시 : resolves #1 -->

resolves #74

gmlyb1 committed:
EmotionRanking
- 마크업 접근성 개선 refactor

devHyeon99 commented on src/components/molecule/EmotionRanking.jsx:
```diff
@@ -3,21 +3,9 @@ import { EmotionRankCard } from '@/components';
 
 const EmotionRanking = () => {
   const rankingsData = [
```
컴포넌트 외부로 이동

devHyeon99 commented on src/components/molecule/EmotionRanking.jsx:
```diff
@@ -34,36 +22,40 @@ const EmotionRanking = () => {
       <div className="flex justify-between">
         {rankingsData.map((rankItem) => (
-          <div
+          <article
```
article 사용시 제목이 있어야해서 제목을  추가하고 rankItem.mood sr-only 사용해서 숨기면 될거 같아요

devHyeon99 commented on src/components/molecule/EmotionRanking.jsx:
```diff
@@ -34,36 +22,40 @@ const EmotionRanking = () => {
               rank={rankItem.rank}
             />
-          </div>
+            <div id={`rank-item-${rankItem.rank}`} className="sr-only">
```
이거를 aria관련 속성으로 대체가 가능한지 검토

gmlyb1 committed:
EmotionRanking
-개선사항
1. 정적 데이터 외부 이동
article 사용시 제목 추가 , sr-only 처리
3.aria 관련 속성 처리

gmlyb1 committed:
Emotion Ranking
-<div id={`rank-description-${rankItem.rank}`} className="sr-only">
            순위 {rankItem.rank}위: {rankItem.mood} - 기록 수: {rankItem.count}
          </div> 삭제

devHyeon99 committed:
Merge branch 'develop' into refactor/EmotionRanking
